code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -22,27 +22,26 @@
 	maxHealth = 400
 	health = 400
 	icon_state = "alienq"
-	var/datum/action/small_sprite/smallsprite = new/datum/action/small_sprite()
+	var/datum/action/small_sprite/queen/smallsprite = new/datum/action/small_sprite/queen()
 
-/datum/action/small_sprite
+/datum/action/small_sprite/queen
 	name = "Toggle Giant Sprite - Others will always see you as giant"
 	button_icon_state = "smallqueen"
 	background_icon_state = "bg_alien"
-	var/small = 0
+	var/queen_small = 0

```

Comment:
`var/small = FALSE` (you don't need subtype-specific vars for something that is used in both)